src/css/components/nav-primary.css

```diff
@@ -10,7 +10,7 @@
       margin-bottom: 0;
     }
 
-    &[aria-haspopup="true"] {
+    &[aria-expanded="false"] {
```

Comment:
This issue has been taken care of and merged. It looks like you're starting point was old.